apps/docs/src/local-ts/index.md

```
@@ -0,0 +1,84 @@
+
+### 3. Run the Development Server
+
+```bash
+pnpm tauri dev
+```
+
+Your app will open in a native window with hot reload enabled.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)